### PR TITLE
Update ServerRendering.md

### PR DESCRIPTION
**Pull request by svenanders**
Always use radix when using parseInt. Other options are Number or isNaN, or maybe even better, the ES6 specific Number.isNaN() - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN.

### CODE REVIEW

File: docs/recipes/ServerRendering.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ import { renderToString } from 'react-dom/server'
 function handleRender(req, res) {
   // Read the counter from the request, if provided
   const params = qs.parse(req.query)
-  const counter = parseInt(params.counter) || 0
+  const counter = parseInt(params.counter,10) || 0
```
Comment (gaearon) [inline docs/recipes/ServerRendering.md]: Nit: please put a space after the comma.

File: docs/recipes/ServerRendering.md [hunk 2]
```diff
@@ -242,7 +242,7 @@ function handleRender(req, res) {
   fetchCounter(apiResult => {
     // Read the counter from the request, if provided
     const params = qs.parse(req.query)
-    const counter = parseInt(params.counter) || apiResult || 0
+    const counter = parseInt(params.counter,10) || apiResult || 0
```
Comment (gaearon) [inline docs/recipes/ServerRendering.md]: Same nit here.

File: docs/recipes/ServerRendering.md (hunk 1 above)
Comment (svenanders) [inline docs/recipes/ServerRendering.md]: OK!

File: docs/recipes/ServerRendering.md (hunk 2 above)
Comment (svenanders) [inline docs/recipes/ServerRendering.md]: OK!

**Pull request merged**